FEXCore/Source/Utils/CPUInfo.cpp

```diff
@@ -14,17 +15,30 @@
 namespace FEXCore::CPUInfo {
 #ifndef _WIN32
 uint32_t CalculateNumberOfCPUs() {
-  char Tmp[PATH_MAX];
+  static uint32_t NumCPUs {};
+  if (NumCPUs) {
+    return NumCPUs;
+  }
+
+#define CPUSTRING "/sys/devices/system/cpu/cpu"
```

Comment:
Should this be undef'd after the assignment into the char buffer?

Reply:
undef'd